app/main.py

```diff
@@ -1,3 +1,71 @@
+import json
+
+from datetime import datetime
+from app.home import Home
+from app.customer import Customer
+from app.shop import Shop
+from app.init import init_all_class
+
+
 def shop_trip():
-    # write your code here
-    pass
+    with open("../app/config.json", "r") as info:
```

Comment:
Change file path in order to pass tests
To make sure that the path is correct run tests using the following command in the terminal:
```
pytest . 
```